src/Components/Facility/models.tsx

```diff
@@ -490,6 +490,12 @@ export interface PatientNotesModel {
   created_date: string;
 }
 
+export interface StateType {
+  notes: PatientNotesModel[];
+  cPage: number;
+  totalPages: number;
+}
+
```

Comment:
Either move this to a file that's specific to the use case or use a different name. "StateType" does not describe what the type is exactly for when kept in facility's models.ts file.

Reply:
Renamed it to `PatientNoteStateType`